grouper/fe/templates/group-join.html

```diff
@@ -90,7 +86,6 @@ <h4 class="modal-title">Join Group Agreement</h4>
                     Agree
                 </button>
             </div>
-                </form>
```

Comment:
I'm so confused -- what was this closing?

Reply:
I know, right? I haven't gone through the commit history to figure out what happened.